pkg/reconciler/ingress/lister.go

```diff
@@ -76,8 +73,12 @@ func (l *gatewayPodTargetLister) getIngressUrls(ing *v1alpha1.Ingress, gatewayIp
 	for _, rule := range ing.Spec.Rules {
 		var target status.ProbeTarget
 
-		externalDomains := getExternalDomains(rule, localDomainName)
-		internalDomains := getInternalDomains(rule, localDomainName)
+		var internalDomains, externalDomains []string
+		if rule.Visibility == v1alpha1.IngressVisibilityClusterLocal {
```

Comment:
Do we need this `if`? Both internal and external are the same now.

Reply:
Thank you. It was not necessary and we could handle it the following `RuleIsExternal`.